CHANGELOG.md

```diff
@@ -66,6 +66,9 @@ All notable changes to this project will be documented in this file.
 ## [4.1.0](https://github.com/Backbase/stream-services/compare/4.1.0...4.0.0)
 ### Changed
 - Bumping Service SDK to **16.1.5**
+## [5.0.0](https://github.com/Backbase/stream-services/compare/4.0.0...3.70.0)
```

Comment:
not sure why this is here. I assume it is a duplicate.

@daniel-chiuratto-seabra can you include your PR description in this file here?

Reply:
Done!